lib/vet360/service.rb

```diff
@@ -10,8 +10,8 @@ def initialize(user)
       @user = user
     end
 
-    def perform(method, path, body = nil, headers = {})
-      vet360_id_present!
+    def perform(method, path, body = nil, headers = {}, skip_vet360_id: false)
+      vet360_id_present! unless skip_vet360_id
```

Comment:
That this is necessary seems like a smell a bit, but this handles it well. Maybe a target for a future refactor.